slip.js

```diff
@@ -649,6 +651,7 @@ window['Slip'] = (function(){
         },
 
         onMouseLeave: function(e) {
+            e.stopPropagation();
             if (this.usingTouch) return;
```

Comment:
I think the stop should be after the check, otherwise it will "eat" events it doesn't use anyway.